### ISSUE TITLE
Ints and Arrays not accepted by polygonDB

### ISSUE THREAD

**Issue by NekaouMike** [PolygonDB/PolygonDB#12]
**Describe the bug**
Polygon doesn't record normal ints/arrays into the DB

**To Reproduce**
1. perform a record request where you pass a int or array as the value. *using WS or Polycon

**Expected behavior**
The int or array should be recorded into the DB

**Version**
RockFeller V1.4

**Comment by JewishLewish** [comment]
Can you share code as an example?

**Comment by NekaouMike** [comment]
```js
const ploycon = require('../main.js');

const db = new ploycon("0.0.0.0:8000","better_password");

main()
async function main(){
    return new Promise(async (resolve, reject) => {
    await db.open().catch(err => console.log(err))
    console.log(await db.record("users",0,"age",20).catch(err => console.log(err)))
    console.log(await db.record("users",0,"hobbies",["music","games"]).catch(err => console.log(err)))
    console.log(await db.record("users",0,"verified",false).catch(err => console.log(err)))
    db.close()
    return resolve(console.log("Done!"))
})
}
```
Code for all fail cases. This adds bool because that also fails

**Comment by JewishLewish** [comment]
For these two, 
```js
    console.log(await db.record("users",0,"age",20).catch(err => console.log(err)))
    console.log(await db.record("users",0,"hobbies",["music","games"]).catch(err => console.log(err)))
```

It does Json.Stringify, can you make it print the json.stringify to see what it is outputting.

**Comment by NekaouMike** [comment]
```json
{"action":"record","password":"better_password","dbname":"users","location":"rows.0.age","value":20}
```
I can only get the first one to output but should start to see what is up.

**Comment by NekaouMike** [comment]
That's the full message from polyconJS

**Comment by JewishLewish** [comment]
> I can only get the first one to output but should start to see what is up.

So PolygonDB has a config adjustable called "log" which logs input and output. 
If you can, can you turn it on and show me what PolygonDB is receiving on their end. 

From my end, I am getting:
```c
2023-02-27 11:22:28.6551801 -0800 PST m=+1.873742101 - 
	Address: ...
	Content:{
    "password": "Better_Password",
    "dbname": "ExampleDB",
    "location": "test",
    "action": "record",
    "value": "20"
}
```

**Comment by NekaouMike** [comment]
But no data is recorded into the DB and no response saying it successfully did it is given

**Comment by JewishLewish** [comment]
I think the problem is with the following code:
```go
// Parses Input that the Websocket would recieve
type input struct {
	Pass   string      `json:"password"`
	Dbname string      `json:"dbname"`
	Loc    string      `json:"location"`
	Act    string      `json:"action"`
	Val    string `json:"value"`
}
```

Value is expecting the data to be string so it cannot marshal properly. 
It's possible to change Val type from string to interface{} so it can accept normal 20

**Comment by NekaouMike** [comment]
Try that and see how it works out

**Comment by JewishLewish** [comment]
Worked.

**Comment by JewishLewish** [comment]
Problem Solved in PolygonDB v1.5
